src/base/init/init.c

```diff
@@ -417,13 +416,11 @@ void low_mem_init(void)
   /* establish alias access for int15 */
   register_hardware_ram_virtual('X', LOWMEM_SIZE + HMASIZE, phys_rsv,
 	    DOSADDR_REL(ptr2));
-  register_hardware_ram_virtual('U',  DOSADDR_REL(ptr2), phys_rsv,
```

Comment:
But I am still not entirely convinced
this is a good thing to do. Yes, in your
view there is no indetity/non-identity
mappings w/o KVM, and in my - there
is. :)
How about just check the letter in
`register_hardware_ram_virtual()`
and for 'U' (uncommitted) ignore the
phys_addr and use -1?